registry.k8s.io/images/k8s-staging-cluster-api-azure/images.yaml

```diff
@@ -90,6 +90,7 @@
     "sha256:6455a8f1b210dbf012a80d357712a913baf8f026103cc1e6c7f7d4d2949a4f67": ["v1.18.2"]
     "sha256:a9dc05966963030fee4bb2396ddaf2e11fc4edc35ecb18b8cae144ba2267ff6d": ["v1.18.3"]
     "sha256:d4beab4adcd54b43214bb97fffd4c928c3e1eac8fa2bfc22c50664c0a3d56b2c": ["v1.18.4"]
+    "sha256:8895d242a9ee5ce6f064c9bccb08224d833ded136f3bf59a123448900d4c27be": ["v1.18.5"]
```

Comment:
SHA matches with the one present in staging repository. 
https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api-azure/global/cluster-api-azure-controller@sha256:8895d242a9ee5ce6f064c9bccb08224d833ded136f3bf59a123448900d4c27be/details?inv=1&invt=AbxNmw